doc/source/analysis.rst

```diff
@@ -1,5 +1,400 @@
+
+Basic:
+
+- :meth:`Graph.diameter` or :meth:`Graph.get_diameter`
+- :meth:`Graph.girth`
+- :meth:`Graph.radius`
+- :meth:`Graph.average_path_length`
+
+Distributions:
+
+- :meth:`Graph.degree_distribution`
+- :meth:`Graph.path_length_hist`
+
+Connectedness:
+
+- :meth:`Graph.all_minimal_st_separators`
+- :meth:`Graph.minimum_size_separators`
+- :meth:`Graph.feedback_arc_set`
+- :meth:`Graph.cut_vertices` or :meth:`Graph.articulation_points`
+
+Cliques and motifs:
+
+- :meth:`Graph.clique_number` (aka :meth:`Graph.omega`)
+- :meth:`Graph.cliques`
+- :meth:`Graph.maximal_cliques`
+- :meth:`Graph.largest_cliques`
+- :meth:`Graph.motifs_randesu` and :meth:`Graph.motifs_randesu_estimate`
+- :meth:`Graph.g.motifs_randesu_no` counts the number of motifs
+
+Structural:
+
+- :meth:`Graph.edge_connectivity` or :meth:`Graph.edge_disjoint_paths` or :meth:`Graph.adhesion`
+- :meth:`Graph.vertex_connectivity` or :meth:`Graph.cohesion`
+
+Optimality:
+
+- :meth:`Graph.farthest_points`
+- :meth:`Graph.modularity`
+- :meth:`Graph.maximal_cliques`
+- :meth:`Graph.largest_cliques`
+- :meth:`Graph.independence_number` (aka :meth:`Graph.alpha`)
+- :meth:`Graph.maximal_independent_vertex_sets`
+- :meth:`Graph.largest_independent_vertex_sets`
+- :meth:`Graph.mincut`
+- :meth:`Graph.mincut_value`
+- :meth:`Graph.maximum_bipartite_matching` (bipartite graphs)
+
+Other complex measures are:
+
+- :meth:`Graph.assortativity`
+- :meth:`Graph.assortativity_degree`
+- :meth:`Graph.assortativity_nominal`
+- :meth:`Graph.coreness` (aka :meth:`Graph.shell_index`)
+- :meth:`Graph.density`
+- :meth:`Graph.hub_score`
+- :meth:`Graph.transitivity_undirected`
+- :meth:`Graph.transitivity_local_undirected`
+- :meth:`Graph.transitivity_avglocal_undirected`
+- :meth:`Graph.dyad_census`
+- :meth:`Graph.triad_census`
+- :meth:`Graph.reciprocity` (directed graphs)
+- :meth:`Graph.isoclass` (only 3 or 4 vertices)
+
+Boolean properties:
+
+- :meth:`Graph.is_bipartite`
+- :meth:`Graph.is_connected`
+- :meth:`Graph.is_dag`
+- :meth:`Graph.is_directed`
+- :meth:`Graph.is_named`
+- :meth:`Graph.is_simple`
+- :meth:`Graph.is_weighted`
+- :meth:`Graph.has_multiple`
+
+Vertex properties
++++++++++++++++++++
+A spectrum of vertex-level properties can be computed. Similarity measures include:
+
+- :meth:`Graph.similarity_dice`
+- :meth:`Graph.similarity_jaccard`
+- :meth:`Graph.similarity_inverse_log_weighted`
+- :meth:`Graph.diversity`
+
+Structural:
+
+- :meth:`Graph.authority_score`
+- :meth:`Graph.betweenness`
+- :meth:`Graph.bibcoupling`
+- :meth:`Graph.closeness`
+- :meth:`Graph.constraint`
+- :meth:`Graph.cocitation`
+- :meth:`Graph.eccentricity`
+- :meth:`Graph.eigenvector_centrality` aka :meth:`Graph.evcent`
+- :meth:`Graph.pagerank`
+- :meth:`Graph.personalized_pagerank`
+- :meth:`Graph.strength`
+
+Connectedness:
+
+- :meth:`Graph.subcomponent`
+- :meth:`Graph.is_separator`
+- :meth:`Graph.is_minimal_separator`
+- :meth:`Graph.biconnected_components`
```

Comment:
This is not a vertex property (I'd put it under Global)

Reply:
yep